x/sequencer/keeper/params.go

```diff
@@ -19,9 +19,6 @@ func (k Keeper) ValidateParams(ctx sdk.Context, params types.Params) error {
 	if err != nil {
 		return errorsmod.Wrapf(gerrc.ErrInternal, "failed to get base denom: %v", err)
 	}
```

Comment:
I think we should still keep the min bond as a general min bond in the system. we can always set it to 0 if we wish.